src/components/KRadio/KRadio.vue

```diff
@@ -376,51 +370,83 @@ $kRadioDotSize: 6px;
 
       &.has-label, &.has-description {
         .card-content-wrapper {
+          height: auto;
           margin-bottom: var(--kui-space-40, $kui-space-40);
         }
       }
 
       .radio-label {
         @include labelDefaults;
       }
+
+      :deep(#{$kongponentsKongIconSelector}) {
+        color: var(--kui-color-text-neutral, $kui-color-text-neutral) !important;
+      }
```

Comment:
issue: Shouldn't the host app have control over the icon it slots? There would be no way to override this.

Reply:
Figma designs define icon color in different states so this should be controlled by the component.

Reply:
This means that we can NEVER use another color icon within the component? Is that desirable? What if we're using the "kong" icon with gradient, or one of the flag icons down the road?

Reply:
It can't be overwritten using `id` selector.
```
...
<KongIcon id="specific-color-icon" />
...
#specific-color-icon {
  color: $kui-color-text-success !important;
}
```

![Screen Shot 2023-11-20 at 10 30 28 AM](https://github.com/Kong/kongponents/assets/36751160/2d0eeb57-c829-4b7e-a833-acf73f3807da)

Reply:
Doesn't seem ideal IMO; that's the whole point of the `color` prop on the icons?

Reply:
Fixed now